src/signals/incident-management/containers/IncidentDetail/components/MetaList/components/ChangeValue/index.test.js

```diff
@@ -10,22 +10,18 @@ import ChangeValue from './index';
 jest.mock('shared/services/list-helper/list-helper');
 
 
-const expectInitialState = async ({ getByTestId, queryByTestId, findByTestId }) => {
-  const editButton = await findByTestId('editButton');
-  const valuePath = getByTestId('valuePath');
+const expectInitialState = async ({ queryByTestId, findByTestId }) => {
+  const editButton = await findByTestId('editMockTypeButton');
 
-  expect(valuePath).toBeInTheDocument();
   expect(editButton).toBeInTheDocument();
   expect(queryByTestId('changeValueForm')).not.toBeInTheDocument();
 };
 
 const expectEditState = async ({ queryByTestId, findByTestId }) => {
-  const editButton = queryByTestId('editButton');
-  const valuePath = queryByTestId('valuePath');
+  const editButton = queryByTestId('editMockTypeButton');
```

Comment:
The construction of the mock name is difficult to understand at the first glance because the name is constructed dynamically. I would choose a more straight forward solution where the test-id name is either explicit set 
on the button, or it can be selected through the meta-list selector.

Reply:
The issue here is that when I use a generic test id value, there is no way to distinguish between the elements in e2e tests. That is why their values are dynamically generated based on their context. In the case of the `ChangeValue` component, that is the incoming data.